docs/architecture.md

```diff
@@ -42,4 +42,4 @@ subresource of it's `Cluster` resource.
 ## Scalability
 
 Fleet is designed to scale up to 1 million clusters. There are more details to come here on how we expect to scale
-a Kubernetes controller based architecture to 100's of millions of objects and beyond.
\ No newline at end of file
+a Kubernetes controller based architecture to 100's of millions of objects and beyond.
```

Comment:
nitpick: Perhaps we could change this to the following?
```
Kubernetes, controller-based
```

Reply:
No worries @nickgerace, this whole PR is based on nit pickiness 😆